Makefile

```diff
@@ -64,7 +64,7 @@ GOLANGCI_VERSION = 1.16.0
 PROTOC_RELEASE_BASE = https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/protoc-$(PROTOC_VERSION)
 GO = GO111MODULE=on go
 # Defines the absolute local directory of the open-match project
-REPOSITORY_ROOT := $(dir $(abspath $(MAKEFILE_LIST)))
+REPOSITORY_ROOT := $(realpath $(dir $(abspath $(MAKEFILE_LIST))))
```

Comment:
What's the reason for this? It was causing issues in Windows.

Reply:
without realpath the REPOSITORY_ROOT is something ending with a slash e.g. GoogleCloudPlatform/open-match/, which makes the local env variables defined in the Makefile becomes open-match//build - a path with two slashes. What is the issue in Windows?

Reply:
It double adds the path in Windows so I had to remove it.